src/SwedbankPay.Sdk/SwedbankPayHttpClient.cs

```diff
@@ -0,0 +1,136 @@
+﻿namespace SwedbankPay.Sdk
+{
+    using Newtonsoft.Json;
+    using Newtonsoft.Json.Linq;
+    using Newtonsoft.Json.Serialization;
+
+    using RestSharp;
```

Comment:
Please don't use `RestSharp`. Microsoft's `HttpClient` provides a much better abstraction for what this SDK is supposed to do.